README.md

```diff
@@ -99,8 +99,8 @@ development. Other versions may or may not work as well.
 
 If you're using a newer version of Ubuntu, you can install these 
 libraries from aptitude. If you're using Mac, or an older Linux/BSD, there 
-are build scripts that download and compile the libraries from source. 
-Follow the instructions for the platform you're compiling on below.
+are build scripts that download and compile the libraries from the source. 
+Follow the instructions for the platform you're compiling below.
```

Comment:
Are we compiling the platform? I think the current wording is better here, we are compiling OpenSCAD _on_ the platform of choice.

Reply:
Also, I think "compiling from source" is a common term in software dev, even though it may not be 100% correct language wise (not an expert on that :) )

Reply:
"Follow the instructions below for the platform you're compiling on" might be better, avoiding the strange "on below" sequence.

Reply:
Thanks for the comments, this helps me as non-native speaker. @wenqingl would you mind updating the PR with the suggested changes (or give an argument for keeping the original version)?